test/index.test.js

```diff
@@ -1117,4 +1117,225 @@ describe('index', () => {
             });
         });
     });
+
+    describe('_addWebhook', () => {
+        const oauthToken = 'oauthToken';
+        const scmUri = 'bitbucket.org:arnoldWeber/wyattLib:mybranch';
```

Comment:
Yes, we're aware that the API requires it. We know the request will be malformed if the curly braces are omitted. What we don't know is why record it when the curly braces are omitted?